flocker/node/test/test_deploy.py

```diff
@@ -1038,7 +1041,7 @@ def test_discover_manifestation_with_size(self):
         )
         d = api.discover_state(self.EMPTY_NODESTATE)
 
-        self.assertItemsEqual(
```

Comment:
This change is necessary, since `Manifestation` was a mapping (so `assertItemsEqual` would do something with it) but now it isn't. That is, the tests started failing with the change to a `PClass` here.